web/modules/custom/template_suggestion/template_suggestion.module

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * @file
+ * Hook implementations for the Template suggestion module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Entity\Entity\EntityViewDisplay;
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Field\FormatterInterface;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_field_formatter_third_party_settings_form().
+ *
+ * Allow adding template suggestions for each field.
+ */
+function template_suggestion_field_formatter_third_party_settings_form(FormatterInterface $plugin, FieldDefinitionInterface $field_definition, $view_mode, $form, FormStateInterface $form_state) {
```

Comment:
Can you briefly explain why this cannot remain in joinup profile or even better the Joinup theme, since all templates are stored there?
We don't test appearance anywhere in the tests and you have said in the past that view modes should reflect the content needed to be viewable so that a functionality works on any theme (meaning that fields and links and texts that should be shown in a page should be visible in any theme even if they are ugly). In that context, this is not needed as an independent module functionality and can easily integrate to joinup_theme.

Reply:
Oh ok.. I see it now.. It is the schema that breaks :/ Quite unfortunate to be honest.. :/

Reply:
Yeah it was a small mistake in the past which grew bigger as the code base expanded, classic case of technical debt :)

Reply:
Would this perhaps be something to contribute? It is pretty handy even though it is small and simple.